resources/public/api/conf/1.0/docs/overview.md

```diff
@@ -1,9 +1,11 @@
-The Self Assessment Accounts API allows a developer to:
+This API allows a developer to:
 
-* Retrieve the overall liability broken down into overdue, payable and pending amounts
-* Retrieve a list of charges and payments for a given date range
-* Retrieve more detail about a specific transaction
-* Retrieve a list of charges made to an account for a given date range
-* Retrieve the history of changes to an individual charge
-* Retrieve a list of payments for a given date range
-* Retrieve the allocation details of a specific payment against one or more liabilities
\ No newline at end of file
+* retrieve the overall liability broken down into overdue, payable and pending amounts
+* retrieve a list of charges and payments for a given date range
+* retrieve more detail about a specific transaction
+* retrieve a list of charges made to an account for a given date range
+* retrieve the history of changes to an individual charge
+* retrieve a list of payments for a given date range
+* retrieve the allocation details of a specific payment against one or more liabilities
+
+For information on how to connect to this API [see the Income Tax MTD end-to-end service guide](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/).
```

Comment:
small thing, but I know you might be asked to; will want to remove the full stop at the end of this link.